public/packs/manifest.json

```diff
@@ -1,13 +1,13 @@
 {
-  "application.js": "/packs/js/application-53dd82e6f5de6f236733.js",
-  "application.js.map": "/packs/js/application-53dd82e6f5de6f236733.js.map",
+  "application.js": "/packs/js/application-4c20dc97a6c83c8ad194.js",
```

Comment:
can you remove manifest.json file from git? add it to gitignore as well

Reply:
Removed manifest.json

Reply:
manifest.json is added to git ignore and changes are saved